Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,35 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// You can bypass SSL peer verification altogether (not advised)
+    public static var verifyPeerSSLCertificate = true
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the path
+    /// provided by setting URLSession.sslCertificateAuthorityFile to a URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL {
+        set(value) {
+            verifyPeerSSLCertificate = true
```

Comment:
Well, thanks for bearing with me. I do like this design better. No need for the `fatalError` guard since you've documented the behavior. In fact, I think I like it better without the additional guard, as the guard itself is triggered or not depending on the order in which the properties are set :P